README.md

```diff
@@ -0,0 +1,162 @@
+# Placed Affiliate SDK (Android) v1.20
+#### Last Updated: August 23, 2016
+  
+## Integrating your application with the Placed Monetization SDK
+  
+### Adding the dependency to your project
+
+1. Add the following to your **root** `build.gradle` file:
+
+```
+allprojects {
+    repositories {
+        ...
+
+        maven { url "https://raw.githubusercontent.com/placed/android-placed-sdk/master/repository" }
```

Comment:
No, because that will point to the github webpage for that directory. We need the raw file directory.